livekit/src/rtc_engine/rtc_session.rs

```diff
@@ -518,6 +518,7 @@ impl SessionInner {
                         })
                         .unwrap(),
                         target: target as i32,
+                        r#final: false,
```

Comment:
unsure about this one, lmk if it needs to be taken from somewhere

Reply:
seems like it is unused on client-sdk-js
what you can do is to put ..Default::default() instead of false.